roadmap/5_developer_experience/pending/dfx_analytics.md

```diff
@@ -0,0 +1,9 @@
+---
+title: DFX analytics
+links:
+  Forum Link: https://forum.dfinity.org/t/dfx-telemetry-proposal/15834
+  Proposal:
+eta: Q4 22
+is_community: false
+---
+We are working on adding telemetry collection to the Internet Computer SDK to improve the platform's developer experience based on insights derived from behavioral data. 
```

Comment:
```suggestion
We are working on adding (purely opt-in) telemetry collection to the Internet Computer SDK to improve the platform's developer experience based on insights derived from behavioral data. 
```
I think it's worth pointing out wherever we can that we want to do a non-scary version of telemetry.